spec.md

```diff
@@ -377,24 +377,32 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
+
+A Platform will find it useful for the Broker to define exactly what is expected
+when making requests inside opaque input request objects and unspecified output
```

Comment:
This paragraph reads funny to me. How about:
```
Platforms MAY be able to provide a better user experience if Service Brokers can programmatically define the inputs and outputs of requests using JSON Schema Object definitions.
```

Reply:
I like it. Added.